src/main/java/com/hedera/cli/hedera/crypto/CryptoTransferMultiple.java

```diff
@@ -258,7 +254,6 @@ private void saveTransactionToJson(TransactionRecord record) {
         AccountId accountId;
         String acc, amt;
         Map<Integer, Recipient> map = new HashMap<>();
-        // TODO: unused local variable?
         long sum = 0;
```

Comment:
This is an unused local variable. Which potentially means that there is unnecessary complexity from line 257 to line 277.

please fix.

Reply:
removed unused variable, no added complexity. resolved in other related branch accountDelete

Reply:
```suggestion
        long sum = 0;
```

The `sum` variable is the unused variable.